ui/webpack.config.js

```diff
@@ -98,6 +98,7 @@ module.exports = {
 	],
 	devServer: {
 		port: 8083,
+        https: true,
```

Comment:
Wrong indents

Reply:
yes the entire file was indented with tabs. changed.